invokeai/frontend/web/package.json

```diff
@@ -1,11 +1,15 @@
 {
-  "name": "invoke-ai-ui",
-  "private": true,
-  "version": "0.0.1",
+  "name": "@invoke-ai/invoke-ai-ui",
+  "version": "0.0.1-dev5",
+  "publishConfig": {
+    "access": "restricted",
+    "registry": "https://npm.pkg.github.com"
+  },
   "scripts": {
     "prepare": "cd ../../../ && husky install invokeai/frontend/web/.husky",
     "dev": "concurrently \"vite dev\" \"yarn run theme:watch\"",
     "build": "yarn run lint && vite build",
+    "build:package": "vite build --mode=package",
```

Comment:
How about something like this:
```json
    "build:spa": "vite build",
    "build:package": "vite build --mode=package",
    "build": "yarn run lint && vite build && vite build --mode=package",
```

Reply:
I wasn't sure if we wanted the artifacts needed for `build:package` in the main build. Maybe would bloat it a bit? But I guess if they are all getting tracked regardless it doesn't really matter.

Reply:
I guess in theory I could actually .gitignore the ones I'm generating if bundle size is a concern

Reply:
Don't think we care about repo size at this point, installing the whole thing is like 25GB anyways >_< I don't have a strong opinion here just was thinking out loud